local-tests/tests/wrapped-keys/testBatchGeneratePrivateKeys.ts

```diff
@@ -0,0 +1,154 @@
+import { log } from '@lit-protocol/misc';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+import { api } from '@lit-protocol/wrapped-keys';
+import { getPkpSessionSigs } from 'local-tests/setup/session-sigs/get-pkp-session-sigs';
+import nacl from 'tweetnacl';
+import bs58 from 'bs58';
+import { ethers } from 'ethers';
+import { BatchGeneratePrivateKeysActionResult } from '../../../packages/wrapped-keys/src/lib/types';
+
+const { batchGeneratePrivateKeys, exportPrivateKey } = api;
+
+async function verifySolanaSignature(
+  solanaResult: BatchGeneratePrivateKeysActionResult,
+  solanaMessageToSign
+) {
+  const {
+    signedMessage: { signature },
+    generatedPrivateKey: { generatedPublicKey },
+  } = solanaResult;
+  const signatureIsValidForPublicKey = nacl.sign.detached.verify(
+    Buffer.from(solanaMessageToSign),
+    bs58.decode(signature),
+    bs58.decode(generatedPublicKey)
+  );
+
+  console.log({ signatureIsValidForPublicKey, signature });
+  if (!signatureIsValidForPublicKey) {
+    throw new Error(
+      `signature: ${signature} doesn't validate for the Solana public key: ${generatedPublicKey}`
+    );
+  }
+}
+async function verifyEvmSignature(
+  litNodeClient,
+  evmResult,
+  messageToSign,
+  pkpSessionSigs
+) {
+  function verifyMessageSignature() {
+    try {
+      return ethers.utils.verifyMessage(
+        messageToSign,
+        evmResult.signedMessage.signature
+      );
+    } catch (err) {
+      throw new Error(
+        `When validating signed Ethereum message is valid: ${err.message}`
+      );
+    }
+  }
+
+  const { decryptedPrivateKey } = await exportPrivateKey({
+    litNodeClient,
+    network: 'evm',
+    id: evmResult.generatedPrivateKey.id,
+    pkpSessionSigs,
+  });
+
+  const recoveredAddress = verifyMessageSignature();
+
+  const wallet = new ethers.Wallet(decryptedPrivateKey);
```

Comment:
good call -- I actually took this assertion pattern from the original `testEthereumBroadcastTransactionGeneratedKey` test that you wrote, to keep things consistent with our existing tests. Should we go ahead and modify them both to avoid exporting the privateKey for either one

Reply:
I think we could make this change as it's a minor one? @MaximusHaximus

Reply:
:+1: Modified this test file -- but I was mistaken about it coming from `testEthereumBroadcastTransactionGeneratedKey` and the place it did come from was slightly different, so i left that one alone in the interest of time since this is kind of a :nit: and the existing test has been GTG